src/Database/Driver.php

```diff
@@ -190,16 +189,14 @@ public function version(): string
      *
      * @return \PDO
      */
-    public function getConnection(): PDO
+    public function getPDO(): PDO
```

Comment:
Nope, it surely shouldn't be. But I couldn't change it because `Fixture/SchemaLoader` uses the `PDO` instance directly to run the query. So we'll need to figure out how to deal with that and then this method can be made protected.

Reply:
Sounds like some goofball found an accidentally public API and used it. :trollface: We could probably fix that usage if we have another way to do an unprepared query. Maybe having a `Driver::exec()` method could be a way forward instead?

Reply:
Done :)